test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -138,6 +138,7 @@
     ("aten::_csr_to_block_csr", datetime.date(2022, 5, 20)),
     ("aten::_weight_norm_cuda_interface", datetime.date(9999, 1, 1)),
     ("aten::_weight_norm_cuda_interface_backward", datetime.date(9999, 1, 1)),
+    ("aten::eye", datetime.date(2022, 9, 30)),
```

Comment:
I suspect that this change is not needed because now you're not modifying the existing signatures. Does the test break without this line?